packages/cli/bin/cli-main.js

```diff
@@ -26,4 +27,13 @@ const opts = minimist(process.argv.slice(2), {
     commands: 'l', // --commands or -l: print commands
   },
 });
+
+const updateNotifier = require('update-notifier');
+// Force version check with `lb4 --version`
+const interval = opts.version ? 0 : undefined;
```

Comment:
curious: what is the behaviour of `updateNotifier` use when this value is `0` or `undefined`? I'm thinking that if `lb4 --version` is run, then it checks right away and if not, then it won't at all.

Reply:
Running `lb4 --version` forces the check immediately with `0`. If the interval is not number, it's default to one day.

Please note `update-notifier` uses a background process and it only prompts upgrade for next run.